net/smcroute/files/smcroute.init

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh /etc/rc.common
+
+START=50
```

Comment:
Why 50 and not 99?
Please use procd instead

Reply:
Ah, that was part of the the original package.

Reply:
I will do that tomorrow.

Reply:
Done.